project_recalculate/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project recalculate
+===================
+
+This module allows to recalculate tasks' start/end dates depending on project's
+start/end dates.
+
+There are two calcutation modes:
+
+* **Date begin**: Task's start/end dates are recalculated from project's date begin
+* **Date end**: Task's start/end dates are recalculated from project's date end
+
+If the project start/end date is changed in the form view, then you have to
+click "Recalculate proyect" button to recalculate all pending tasks
+according to the new date.
+
+
+Configuration
+=============
+
+You can define working calendar at Setting > Technical > Resource > Working time
+Then assign this calendar to a resource (related with an user)
+
+When calculating task dates, this addon will look at user assigned to the task:
+
+* If user assigned:
+    * Search first resource related with this user (normally user is related,
+      only with one resource) and get its working calendar
+    * If not found, search first working calendar of the company
+    * If not found, then every day is workable an work starts at 08:00 and
+      ends at 18:00
+* If not user assigned:
+    * Search first working calendar of the company
+    * If not found, then every day is workable an work starts at 08:00 and
+      ends at 18:00
```

Comment:
Sorry I didn't see that field. IMHO, this would be priority order:

1. Project work calendar
2. Calendar of first resource related with user assigned to the task
3. First calendar related with current company

Are you agreed?